apps/application-system/api/src/app/modules/application/application.configuration.ts

```diff
@@ -0,0 +1,6 @@
+import { registerAs } from '@nestjs/config'
+
+export const applicationConfiguration = registerAs('application', () => ({
+  attachmentBucket: process.env.APPLICATION_ATTACHMENT_BUCKET,
```

Comment:
Any reason for not having this in the environments file?

Reply:
Good question. I guess I missed some discussions about how we configure NestJS modules and was just following the [configuration guide](https://docs.nestjs.com/techniques/configuration).

I'm not sure how I feel about the `Module.register(config)` pattern. It's non-standard, and requires top-level modules to provide configuration all sub-modules, which exposes implementation details, eg with nested modules.

On the other hand, I understand the benefit of having an overview of all configurations/environment-variables in one place.

Reply:
I'll change this to match the convention.